wx/lib/agw/piectrl.py

```diff
@@ -524,8 +533,8 @@ def OnSize(self, event):
     def RecreateCanvas(self):
         """ Recreates the :class:`PieCtrl` container (canvas). """
 
-        self._canvasbitmap = wx.Bitmap(self.GetSize().GetWidth(),
-                                       self.GetSize().GetHeight())
+        self._canvasbitmap = wx.Bitmap(self.GetSize().GetWidth() or 1,
```

Comment:
ummm. I'm not sure what to say here but if you grab the "bottom-right" corner of a frame and resize it to ZERO, then you will get a rather nasty but usual assertion error that something like the bitmap would be 0. 
That is why I put the `or 1` in many cases on the end of it.
Yea, it happens on windows usually....

Reply:
Also sorry I didn't edit the demo...

Reply:
Maybe I thinkx that if we put that statement into the demo it might provide/help some folk some future frustrations, but I cannot be sure because if sys.last_traceback or value is none than that might confuse them also.

Reply:
for example make a def that fires off at ZERO and tells the user what nasty things happen if you didn't do that... @RobinD42 thoughts....

Reply:
We could reference #1764 for example...

Reply:
As far as your request, yes that is fine as it will end up on 1 if failure, but just to note it is longer, a bit more descriptive, which might be confusing, and lastly, I benchmarked the lib code for performance, but yes `max(1, self.GetSize().GetWidth())` is fine by me in the Demo file... tho it is not fastest.
First time I got anyone to do the slow way.
Implementing commitREQz now...

Reply:
kk. REQz filled unless you have other requests. Please test before merging. :)

Reply:
Readability counts+